test/integration/simple_project/controls/gcloud.rb

```diff
@@ -51,12 +55,51 @@
       end
     end
 
-    describe "enableFlowLogs" do
-      it "should be true" do
-        expect(data).to include(
-          "enableFlowLogs" => true
-        )
+    it "enableFlowLogs should be true" do
```

Comment:
I believe the `enableFlowLogs` param is deprecated on the API, so we can probably drop this and simply test for the log config directly.

Reply:
No problem, will do!